test/registry_test.go

```diff
@@ -0,0 +1,145 @@
+// Copyright 2021 CloudWeGo Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package test
```

Comment:
In the Go world, we don't use a test package to write all tests, just put the test files into the package that you want to test.

like: https://github.com/cloudwego/kitex/tree/develop/pkg/acl package

Reply:
I think you could just remove replace it